apps/emqx_dashboard/src/emqx_swagger.erl

```diff
@@ -0,0 +1,43 @@
+-module(emqx_swagger).
```

Comment:
this module name is probably not well enough namespaced

Reply:
rename to emqx_dashboard_swagger_util  or emqx_dashboard_swagger_ref?

Reply:
`emqx_dashbaord_swagger` should be enough.